### PR TITLE
Updated side panel to show provider name and generate list of campusses (ford3-169)

### LINKED ISSUE #169: Update sidebar to show current campus name
image

### PR DESCRIPTION
**Pull request by Christiaanvdm**
#This closes #169 

image

### CODE REVIEW

**Comment by pierrealixt** [comment]
@Christiaanvdm Can you fix the style as well?

**Comment by Christiaanvdm** [comment]
NewGIF

File: django_project/ford3/views/provider.py [hunk 1]
```diff
@@ -64,16 +64,21 @@ def edit_provider(request, provider_id):
...
+    campus_data = campus_query.values('name', 'id')
     provider_name = campus_query.values('provider_name')[0]['provider_name']
 
     form_data['campus_list'] = list(campus_data)
     form_data['provider_name'] = str(provider_name)
-    return render(request, 'provider.html',
-                  {'form_data': form_data})
+
+    context['form_data'] = form_data
+    context['provider'] = {}
```
Comment (pierrealixt) [inline django_project/ford3/views/provider.py]:
```
context['provider'] = {
  campus: campus_data
  id: provider_id
}
```

File: django_project/core/base_templates/side_navbar.html [hunk 2]
```diff
@@ -64,10 +64,11 @@
 		background-color: rgba(255, 255, 255, 0.13);
```
Comment (pierrealixt) [inline django_project/core/base_templates/side_navbar.html]: CSS in the html?

File: django_project/core/base_static/css/stylesheet.css [hunk 3]
```diff
@@ -697,3 +706,8 @@ input[type='email']:hover::placeholder {
     background-size: contain;
     display: inline-flex;
 }
+
+.noplr {
```
Comment (pierrealixt) [inline django_project/core/base_static/css/stylesheet.css:755]: you could use bootstrap class: `pl-0 pr-0`
https://getbootstrap.com/docs/4.3/utilities/spacing/

File: django_project/core/base_static/css/side_navbar.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
+.wrapper {
```
Comment (pierrealixt) [inline django_project/core/base_static/css/side_navbar.css]: :/ let's call this file just `sidebar.css`

File: django_project/core/base_static/css/stylesheet.css (hunk 3 above)
Comment (Christiaanvdm) [reply django_project/core/base_static/css/stylesheet.css:755]: Those classes didn't work, unsure why.

**Review by pierrealixt — APPROVED** [review]
LGTM

**Pull request merged**